semgrep/semgrep/commands/scan.py

```diff
@@ -151,6 +151,12 @@ def convert(
 @click.command(context_settings=CONTEXT_SETTINGS)
 @click.argument("target", nargs=-1, type=click.Path(allow_dash=True))
 @click.help_option("--help", "-h", help=("Show this message and exit."))
+@click.option(
+    "--apply",
+    is_flag=True,
+    help=("Print a list of job postings at r2c."),
+    # hidden=True, # Not sure if this should be hidden
```

Comment:
```suggestion
    hidden=True,
```

yeah I think this makes most sense as a hidden easter egg 👍